plugins/bigquery/dbt/include/bigquery/macros/adapters.sql

```diff
@@ -26,6 +26,20 @@
 
 {%- endmacro -%}
 
+{% macro table_options() %}
+  {%- set raw_persist_docs = config.get('persist_docs', {}) -%}
+  {%- set raw_persist_relation_docs = raw_persist_docs.get('relation', false) -%}
+
+  {%- if raw_persist_docs is {} -%}
+    {{ return('') }}
+  {% endif %}
+
+  OPTIONS(
+    {% if raw_persist_relation_docs -%}
+      description={{ model.description }}
```

Comment:
We'll need to quote the model description and likely escape it too! One way to do that might be to use the `tojson` filter?
```
description={{ model.description | tojson }}
```

which should render out to
```
description="this is a \"description\" for a \"model\""
```

dbt should in general have better mechanisms for sql escaping -- let me have a think about this one